engine/src/main/java/org/hibernate/search/engine/metadata/impl/AnnotationMetadataProvider.java

```diff
@@ -1040,7 +1040,9 @@ private void initializeMemberLevelAnnotations(String prefix,
 		);
 		checkForContainedIn( member, typeMetadataBuilder, parseContext );
 
-		numericFields.validate();
+		if ( !parseContext.skipFieldBridges() ) {
```

Comment:
@Sanne it's probably something we should discuss for 6. The annotation metadata provider magic became a little too magic and obscure lately with Es and obscure bug fixing.